products/emberjs.md

```diff
@@ -27,10 +27,17 @@ auto:
 # - support(x) = release(x+1)
 # - eol(x) = release(x+1)
 releases:
--   releaseCycle: "5.4"
-    releaseDate: 2023-10-30
+-   releaseCycle: "5.5"
+    releaseDate: 2023-12-17
     support: true
     eol: false
+    latest: "5.5.0"
+    latestReleaseDate: 2023-12-17
+
+-   releaseCycle: "5.4"
+    releaseDate: 2023-10-30
+    support: 2023-12-17
+    eol: 2023-12-17
```

Comment:
```suggestion
-   releaseCycle: "5.4"
    releaseDate: 2023-10-30
    lts: 2023-12-17
    support: 2024-08-25
    eol: 2024-12-29
```

On https://blog.emberjs.com/ember-released-5-5/ it says that 5.4 has been promoted to LTS.

Reply:
Good catch :+1:

Reply:
On https://emberjs.com/releases/lts/ there are other dates for 5.4:
```suggestion
-   releaseCycle: "5.4"
    releaseDate: 2023-10-30
    lts: 2023-12-17
    support: 2024-08-19
    eol: 2024-12-23
```
@marcwrobel Agree?

Reply:
Yes, documented dates are better than computed ones.